karaf 4.4.9 - missing requirement [org.ops4j.pax.logging.pax-logging-api/2.3.0] osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"

Unable to build karaf via assemby plugin, getting this exception:
```
[ERROR] Failed to execute goal org.apache.karaf.tooling:karaf-maven-plugin:4.4.9:assembly (default-assembly) on project karaf-assembly: Unable to build assembly: Unable to resolve root: missing requirement [root] osgi.identity; osgi.identity=framework; type=karaf.feature; version=4.4.9; filter:="(&(osgi.identity=framework)(type=karaf.feature)(version>=4.4.9))" [caused by: Unable to resolve framework/4.4.9: missing requirement [framework/4.4.9] osgi.identity; osgi.identity=org.ops4j.pax.logging.pax-logging-api; type=osgi.bundle; version="[2.3.0,2.3.0]"; resolution:=mandatory [caused by: Unable to resolve org.ops4j.pax.logging.pax-logging-api/2.3.0: missing requirement [org.ops4j.pax.logging.pax-logging-api/2.3.0] osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"]] -> [Help 1]
```

The only thing I came up with is that the pax-logging-api/2.3.0 has this requirement:
```
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=11))
```
and the previous one (2.2.8) had:
```
Require-Capability: osgi.ee;filter:="(&(osgi.ee=JavaSE)(version=1.8))"
```

and the values in config.properties / eecap-xxx have 11.0  instead of 11; but I'm unsure whether that matters.

--- a/tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/AssemblyMojo.java
+++ b/tooling/karaf-maven-plugin/src/main/java/org/apache/karaf/tooling/AssemblyMojo.java
@@ -393,9 +393,9 @@ public class AssemblyMojo extends MojoSupport {
      *     <li><code>org.osgi.framework.system.packages</code></li>
      *     <li><code>org.osgi.framework.system.capabilities</code></li>
      * </ul>
-     * <p>Valid values are: 1.6, 1.7, 1.8, 9
+     * <p>Valid values are: 1.8, 11, 17, 22
      */
-    @Parameter(defaultValue = "1.8")
+    @Parameter(defaultValue = "11")
     protected String javase;
 
     /**

